bin/pycbc_split_inspinj

```diff
@@ -2,28 +2,44 @@
 
 # Copyright (C) 2014 Andrew Lundgren
```

Comment:
I don't even remember writing this, so no objections as long as it still works in the main workflows where it's used.